script.sharethetv/addon.xml

```diff
@@ -14,4 +14,5 @@
     <disclaimer lang="en">New users will need to create an account at sharethe.tv before using.</disclaimer>
     <platform>all</platform>
   </extension>
+  <broken>Website no longer exists</broken>
```

Comment:
@KarellenX broken tag must be part of the `<extension point="xbmc.addon.metadata">` element